Subject Interaction Suite/common/treaty_articles/z03_support_independence.txt

```diff
@@ -76,30 +78,36 @@
 				save_temporary_scope_as = actor
 			}
 			custom_tooltip = {
-				text = "DIPLOMATIC_PACT_REQUIREMENT_MODERATE_LIBERTY_DESIRE"
+				text = "DIPLOMATIC_PACT_REQUIREMENT_HIGH_LIBERTY_DESIRE"
 				OR = {
 					scope:source_country = {
 						is_subject = no
 					}
 					scope:source_country = {
-						NOT = { overlord = scope:target_country.overlord }
+						overlord = scope:target_country.overlord
 					}
-					# scope:target_country.liberty_desire >= define:NDiplomacy|WAR_GOAL_INCREASE_AUTONOMY_MIN_LIBERTY_DESIRE
-					# Seriously people, the point of supporting independence is to encourage a subject to seek autonomy, not just to back a subject who already wants it. Geopolitics people...
-					scope:target_country.liberty_desire >= define:NDiplomacy|WAR_GOAL_INFAMY_LOW_LIBERTY_DESIRE_THRESHOLD
+					scope:source_country.liberty_desire >= 25 #offering country should not be about to lose its own independence 
```

Comment:
```suggestion
					scope:source_country.liberty_desire >= define:NDiplomacy|WAR_GOAL_INFAMY_LOW_LIBERTY_DESIRE_THRESHOLD # Keep using the define in case we/another mod changes the threshold - #offering country should not be about to lose its own independence 
```